content/updates/2018-02.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 2018.2.0
+date: 2018-03-07
+hidden: true
+image: TODO
+---
+
+# 2018.02 Changes
+
+Vision:
+
+- Completely redo vision to greatly increase its accuracy (now all Cartesian co-ordinates are within 15% of their distance from the camera, all angles are within 5 degrees.)
+- Greatly improve marker detection, it is now much less dependent on lighting conditions.
+
+Logging:
+
+- Add a version number message on start-up.
+- Improve robot logging so you know which messages come from our code, and which comes from yours.
```

Comment:
Please could you reword this sentence, it doesn't sound quite right. Particularly:
>which messages **come** from our code, and which **comes** from yours.

EDIT: Perhaps use "where the messages come from"

Reply:
fixed in c85e48aa0d696baa1587cc95bfaec5516d87baeb